cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -74,8 +74,15 @@ def setup_authentication():
     set_state('authentication.setup')
 
 
+@when('kube_master_components.installed')
+def set_app_version():
+    ''' Declare the application version to juju '''
+    version = check_output(['kube-apiserver', '--version'])
+    hookenv.application_version_set(version.split(b' ')[-1].rstrip())
```

Comment:
Include a comment with the __current__ version output so if this split/rstrip ever breaks we can understand what changed.